scrum/tools/trello/card.py

```diff
@@ -14,6 +14,7 @@ def __init__(
             self,
             card_id,
             story_points,
+            short_link='',
```

Comment:
I made one change here to make the short_link optional, as it was breaking the testing for the card class. Let me know if that's in line with what you wanted